server.js

```diff
@@ -80,6 +80,34 @@ app.get('/resize_crop', function(req, res, next){
 	});
 });
 
+app.get('/crop/:w/:h/:x/:y/:u', function(req, res, next){
+	var u =  decodeURIComponent(req.params.u) // URL
+		, w =  req.params.w // width
+		, h =  req.params.h // height
+		, x =  req.params.x // x-offset
+		, y =  req.params.y // y-offset
+		;
+	if (!u || !w || !h || !x || !y) res.send(400, {error: crop_message});
```

Comment:
It would be good to extract this code block to a common function shared with the `/crop` route for re-use. That allows the easy definition of the 2 route signatures and different route param setup but re-uses the 1 code block for running the crop.